src/components/Header.js

```diff
@@ -1,19 +1,26 @@
 import { NavLink } from 'react-router-dom';
+import avatar from '../avatar.svg';
 
 const Header = () => (
-  <header>
-    <h2>Bookstore CMS</h2>
-    <nav>
-      <ul>
-        <li>
-          <NavLink to="/">BOOKS</NavLink>
-        </li>
-        <li>
-          <NavLink to="/categories">CATEGORIES</NavLink>
-        </li>
-      </ul>
-    </nav>
-  </header>
+  <>
+    <header>
+      <nav className="navbar">
+        <h2 className="logo">Bookstore CMS</h2>
+        <ul>
+          <li className="books-link">
+            <NavLink to="/">BOOKS</NavLink>
+          </li>
+          <li className="categories">
+            <NavLink to="/categories">CATEGORIES</NavLink>
```

Comment:
- [ ] As per Zeplin design, you have not respected the text color of your Navbar tabs. Kindly make the color of Books and Categories same as in the Zeplin design